lib/active_merchant/billing/gateways/braintree_blue.rb

```diff
@@ -533,7 +533,10 @@ def create_transaction_parameters(money, credit_card_or_vault_id, options)
           :order_id => options[:order_id],
           :customer => {
             :id => options[:store] == true ? "" : options[:store],
-            :email => scrub_email(options[:email])
+            :email => scrub_email(options[:email]),
+            :phone => options[:phone] || options.dig(:billing_address, :phone),
+            :first_name => options[:billing_address][:name].split(/\s+/, 2)[0], # getting first name
+            :last_name => options[:billing_address][:name].split(/\s+/, 2)[1] # getting all names after first
```

Comment:
After looking at this again I think we should split out any extra names into the first name field and have the last name contain only the last name in the case of multiples.